Jenkinsfile

```diff
@@ -42,8 +42,13 @@ webappPipeline {
     postReleaseStep = {
         sshagent(credentials: [constants.credentials.github.inin_dev_evangelists]) {
             sh("""
+                # tag the version
                 git tag v${version}
                 git push origin --tags
+                # patch to prep for the next version
+                npm version patch --no-git-tag-version
+                git commit -am "Prep next version"
+                git push origin HEAD:master --tags
```

Comment:
How does this not result in a release loop?

Reply:
I thought this was fixed to prevent the loop? I _think_ Cheely is doing something similar. And I _think_ this is the code that will ignore the build: [webappPipeline.groovy#L262](https://bitbucket.org/inindca/pipeline-library/src/d6134bc496c6c23e00055aae733a5f4ee4b7ba3c/vars/webappPipeline.groovy#lines-262)

Reply:
awesome. just making sure.